tests/plotting/test_charts.py

```diff
@@ -658,6 +654,7 @@ def test_areaplot(chart_2d, area_plot):
     assert np.allclose(area_plot.y2, y2)
 
 
+@pytest.mark.noautofixt
```

Comment:
This is not a built-in fixture. [At the source](https://stackoverflow.com/a/38763328/5067311) it's implemented with the help of
```py
@pytest.fixture(autouse=True)
def autofixt(request):
    if 'noautofixt' in request.keywords:
        return
    print("patching stuff")
```
Currently `pytest` warns
```
tests/plotting/test_charts.py:657
  /home/adeak/pyvista/tests/plotting/test_charts.py:657: PytestUnknownMarkWarning: Unknown pytest.mark.noautofixt - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.noautofixt
```

Reply:
And this looks like it only skips the fixture named `autofixt`, rather than all autofixtures. So we'd have to do this `request` dance on whatever fixture(s) we want to be skipped.

Reply:
Ah yes, this is an artifact of when I was trying to disable the GC validation fixture for tests that are failing it. This was an accidental include.